weld-rendering/app/src/main/java/styler/weld/rendering/adapter/HorizontalAdapter.kt

```diff
@@ -13,7 +13,7 @@ import styler.weld.rendering.viewholder.HorizontalShopViewHolder
 import styler.weld.rendering.viewholder.HorizontalViewHolder
 
 class HorizontalAdapter(
-    private val baseList: List<BaseItem>?,
+    private val baseList: List<HorizontalListItem>?,
     private val type: String
 ) :
```

Comment:
Similar to `WidgetAdapter`, we should need a Factory class here (Will do it in another PR)